src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,
+    Blue,
+    Green,
+    Yellow,
+    White,
+    Black
```

Comment:
Enum values should be in uppercase letters. According to the Java style guide, the names of enum values should be all uppercase letters with words separated by underscores. For example, use RED instead of Red, BLUE instead of Blue, etc.